lib/util/github/graphql/cache-strategies/package-cache-strategy.ts

```diff
@@ -0,0 +1,31 @@
+import * as packageCache from '../../../cache/package';
+import type { GithubDatasourceItem, GithubGraphqlCacheRecord } from '../types';
+import { AbstractGithubGraphqlCacheStrategy } from './abstract-cache-strategy';
+
+/**
+ * Package cache strategy meant to be used for public packages.
+ */
+export class GithubGraphqlPackageCacheStrategy<
+  GithubItem extends GithubDatasourceItem
+> extends AbstractGithubGraphqlCacheStrategy<GithubItem> {
+  load(): Promise<GithubGraphqlCacheRecord<GithubItem> | undefined> {
+    return packageCache.get(this.cacheNs, this.cacheKey);
+  }
+
+  async persist(
+    cacheRecord: GithubGraphqlCacheRecord<GithubItem>
+  ): Promise<void> {
+    const expiry = this.createdAt.plus({
+      days: AbstractGithubGraphqlCacheStrategy.cacheTTLDays,
+    });
+    const ttlMinutes = expiry.diff(this.now, ['minutes']).as('minutes');
```

Comment:
why we use minutes here and seconds on memcache?

Reply:
I started first with memory-based cache and then realized we have to provide minutes for the package cache underlying layer. It's not something that matters from the functionality standpoint.

Reply:
Actually, we don't need this for memory cache, so I've removed it